.travis.yml

```diff
@@ -1,6 +1,17 @@
+sudo: false
+os:
+  - linux
```

Comment:
Can this line be removed? I've always gotten linux test machines when it's not specified.